sway-lib-std/src/message.sw

```diff
@@ -43,3 +43,26 @@ pub fn send_message(recipient: b256, msg_data: Vec<u64>, coins: u64) {
     }
     revert(FAILED_SEND_MESSAGE_SIGNAL);
 }
+
+/// Sends a message `msg_data` of type T to `recipient` with a `coins` amount of the base asset
```

Comment:
Would it be accurate to say that `send_typed_message` is what should be used if you want to be able to index the messages?

Reply:
Yup! Added that as a comment. I also changed `send_typed_message` to look somewhat like `transfer_to_address` in the sense it now takes an `Address` and the recipient is the last argument instead of the first.